component/libfuse/libfuse_handler.go

```diff
@@ -612,12 +611,29 @@ func libfuse_open(path *C.char, fi *C.fuse_file_info_t) C.int {
 	if fi.flags&C.O_SYNC != 0 || fi.flags&C.__O_DIRECT != 0 {
 		log.Err("Libfuse::libfuse_open : Reset flags for open %s, fi.flags %X", name, fi.flags)
 		// Blobfuse2 does not support the SYNC or DIRECT flag. If a user application passes this flag on to blobfuse2
-		// and we open the file with this flag, subsequent write operations wlil fail with "Invalid argument" error.
+		// and we open the file with this flag, subsequent write operations will fail with "Invalid argument" error.
 		// Mask them out here in the open call so that write works.
 		// Oracle RMAN is one such application that sends these flags during backup
 		fi.flags = fi.flags &^ C.O_SYNC
 		fi.flags = fi.flags &^ C.__O_DIRECT
 	}
+	if !fuseFS.disableWritebackCache {
+		if fi.flags&C.O_ACCMODE == C.O_WRONLY {
+			log.Debug("Libfuse::libfuse_open : Flag O_WRONLY (%X) for open %s when write back cache is on.", fi.flags, name)
+			// 	fi.flags = fi.flags &^ C.O_ACCMODE
+			// 	fi.flags = fi.flags | C.O_RDWR
+		}
+
+		if fi.flags&C.O_APPEND != 0 {
+			log.Err("Libfuse::libfuse_open : Unsupported flag O_APPEND (%X) for open %s when write back cache is on. Pass --disable-writeback-cache via CLI or libfuse.disable-writeback-cache: true via config file", fi.flags, name)
```

Comment:
shouldn't this log be in the else statement? since the line after we check if ignore-append is on

Reply:
its still useful in both situations I think